test/functional/p2p-acceptblock.py

```diff
@@ -98,17 +98,13 @@ def __init__(self):
         super().__init__()
         self.setup_clean_chain = True
         self.num_nodes = 2
+        self.extra_args = [[], ["-whitelist=127.0.0.1"]]
 
     def setup_network(self):
         # Node0 will be used to test behavior of processing unrequested blocks
         # from peers which are not whitelisted, while Node1 will be used for
         # the whitelisted case.
-        self.nodes = []
-        self.nodes.append(start_node(0, self.options.tmpdir,
-                                     binary=self.options.testbinary))
-        self.nodes.append(start_node(1, self.options.tmpdir,
-                                     ["-whitelist=127.0.0.1"],
-                                     binary=self.options.testbinary))
+        self.nodes = self.setup_nodes()
```

Comment:
No. The base class `setup_network()` would connect the two nodes, which we don't want in this test, so it is correct to override here. However, there is a bug here - I shouldn't assign `self.nodes` to the return value, which is None.